linux/sound.cpp

```diff
@@ -64,16 +61,9 @@ void CSound::OpenJack ( const bool bNoAutoJackConnect, const char* jackClientNam
     // check sample rate, if not correct, just fire error
     if ( jack_get_sample_rate ( pJackClient ) != SYSTEM_SAMPLE_RATE_HZ )
     {
-        throw CGenErr ( tr ( "The Jack server sample rate is different from "
-                             "the required one. The required sample rate is:" ) +
-                        " <b>" + QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) + " Hz</b>. " +
-                        tr ( "You can "
-                             "use a tool like <i><a href=\"https://qjackctl.sourceforge.io\">QJackCtl</a></i> "
-                             "to adjust the Jack server sample rate." ) +
-                        "<br>" +
-                        tr ( "Make sure to set the "
-                             "Frames/Period to a low value like " ) +
-                        QString().setNum ( DOUBLE_SYSTEM_FRAME_SIZE_SAMPLES ) + tr ( " to achieve a low delay." ) );
+        throw CGenErr ( QString ( tr ( "The JACK server doesn't run at a sample rate of <b>%1 Hz</b>. Please use "
```

Comment:
Should this be "The JACK server isn't..."? Saying "doesn't" implies it never will.

Reply:
Changed this one.